sql/Register.sql

```diff
@@ -0,0 +1,12 @@
+create table user_table(
+	username varchar(30)  unique not null,
```

Comment:
id primary key missing in this table. Id should be auto generated.

Reply:
declared id primary key with auto generated.